libs/api-provider-environment/src/WebbProvider.tsx

```diff
@@ -486,7 +482,8 @@ export const WebbProvider: FC<WebbProviderProps> = ({ children, appEvent }) => {
                 () =>
                   new Worker(
                     new URL(
-                      '@webb-tools/react-environment/arkworks-proving-manager.worker'
+                      '@webb-tools/react-environment/arkworks-proving-manager.worker',
```

Comment:
We should also consider having a way to test out using the same flow for Circom proving (for EVM) on the Substrate side as well. We do have a test showing that those proofs are valid. Possible new task.